FlightAppInSwiftUI/View/HomeView.swift

```diff
@@ -19,16 +24,94 @@ struct HomeView: View {
                         RoundedRectangle(cornerRadius: 20, style: .continuous)
                         .fill(Color.white)
                         .padding(.top, 170)
-                        .ignoresSafeArea()
+                        VStack (alignment: .leading){
+                            Text("Hello")
+                            .font(.system(size: 28, weight: .medium))
+                            Text("Plan your next travel")
+                            .font(.system(size: 17, weight: .regular))
+                        }.frame(minWidth: 0, maxWidth: .infinity, minHeight: 0, maxHeight: .infinity, alignment: .topLeading)
+                            .padding(.top, 70)
+                            .padding(15)
                         ScrollView {
-                            VStack (alignment: .leading){
-                                Text("Hello")
-                                .font(.system(size: 28, weight: .medium))
-                                Text("Plan your next travel")
-                                .font(.system(size: 17, weight: .regular))
-                            }
-                        }.padding(.top, 100)
-                            .ignoresSafeArea()
+                            VStack(spacing: 30) {
+                                Picker("", selection: $flightType) {
+                                    Text("Round Trip").tag(0)
+                                    Text("One Way").tag(1)
+                                }.pickerStyle(.segmented)
+                                ZStack {
```

Comment:
need some refactor here as well

Reply:
This will be fixed after the merge because the final code is in the 'segmentedControl_views' branch